cloudmarker/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Cloudmarker - Cloud security monitoring framework."""
 
 
-__version__ = '0.1.0.dev1'
+__version__ = '0.1.0.dev2'
```

Comment:
The version number has been incremented because I would like to make another dev release of Cloudmarker on PyPI after this PR is merged, so that I can install the new version using `pip3 install cloudmarker` just like a user would.

This would help me to test out the instructions I would be writing for users in the new README currently being worked upon at [userdoc:README.rst](https://github.com/cloudmarker/cloudmarker/blob/userdoc/README.rst).